runtime/manual/basics/react.md

```diff
@@ -1,36 +0,0 @@
----
-title: "Using React with Deno"
----
-
```

Comment:
This page is still referenced in `/runtime/tutorials/index.md`

Reply:
I can't find it

Reply:
nvm, looks like I was in a different branch locally when looking this up.